feat(sessions): :sparkles: add workshop details to intro as reading task (#115)
rostools/github-intro: sessions/introduction.qmd

# Introduction {#sec-introduction}

## :book: Reading task: Workshop details

**Time: \~8 minutes.**

### Learning design {.unnumbered}

> <!-- TODO: Move this over into an include for other websites? -->

Reviewer: Yes 🙌 Seems like a quite general thing that would be very nice to include on the other websites as well

Author: Yea, I will merge this in and then move them over afterwards :tada:

This workshop is designed using the science of learning. Learning is
when new content gets incorporated into long-term storage in the brain
and in a way that it can be later retrieved from storage to be used
again. Some of the best ways to enable learning is to do "active

> learning" by engaging with the material, practice retrieving what was
> just taught, repeatedly be exposed to and retrieve the content, linking

Reviewer: ```suggestion
learning" by engaging with the material, practicing retrieving what was
just taught, repeatedly being exposed to and retrieve the content, linking
```

the new information with what you already know, and explaining it to
someone else.

We use a variety of activities to target the different pathways of
learning. A big part of learning is being engaged in the *process* of
learning, not just learning the content itself. So to help you be
engaged in your own learning, we want to involve you in the reasons
*why* we do the things we do. These are the things we do in this
workshop that we've designed to enable optimal learning:

> -   Having a clear learning goal with a strong narrative throughout the

Reviewer: ```suggestion
-   We have a clear learning goal with a strong narrative throughout the
```
To fit with the later bullet points that start with “we include”, “we have”, etc.

    workshop that ties back to that goal. It's much easier to learn when
    you have a better idea of where you are going, where you are at, and
    what to expect along the way.

> -   The main activities are "code-alongs", where the teacher types on

Reviewer: Ooo, we have mentions of both teachers, instructors, and helpers now. Should this be instructor instead to follow the rostools [guide](https://guides.rostools.org/instructors)?

Author: I need to change the guide. It should be "teacher" now. Like with "course", teacher is a more accurate term to describe what we do. Instructor is more "do this do that", while teacher is more "this is why we do that and this".

>     the computer and explains what it means while the learner types

Reviewer: ```suggestion
    the computer and explains what it means while you type
```
So it fits with the “you” on L37

    along. The connection between using the hands and doing something
    tangible helps reinforce learning. It also helps the teachers to go
    slower since we have to both explain and type things out, which
    helps you learn better. The phrase "less is more" applies strongly
    to learning.

> -   We include several hands-on, practical exercises that reinforce what

Reviewer: ```suggestion
-   We include several hands-on exercises that reinforce what
```
Isn’t having both "hands-on" and “practical” a bit redundant?

    was just taught through retrieval practice and repetition. It gives
    you a chance to engage with the material and the space to try it out

>     on your own so that your brain makes the connections.

Reviewer: ```suggestion
    on your own, so you start to get a real sense of how to do it yourself.
```
Or something like that? I’m thinking that the exercises also aims to help learners feel more “confidant” in how to do the things taught in the workshop by themselves going forward? :)

-   We have discussion activities to encourage linking the new

>     information with what you already know or do in your work and how

Reviewer: ```suggestion
    information with what you already know and do in your work and how
```

    you can apply it. Since humans are social beings, we also tend to
    learn better when there is some social aspect to it. Explaining

>     something to someone else is also one of the best ways to learn! The

Reviewer: ```suggestion
    something to someone else is one of the best ways to learn! The
```

>     quote "while we teach, we learn" is true here.

Reviewer: Do you want to include who said the quote? Like where it’s from?

> -   Reading content is a much better way to consume information than
>     listening to someone talk. This is also why we have minimal to no
>     slides or lectures, as they are forms of information sharing with
>     the lowest rates of learning. You can re-read things you may not
>     understand, go at your pace, and take notes, all of which support
>     learning. We have several reading tasks throughout the workshop. To

Reviewer: I feel like “reading is much better” is a very strong statement that maybe everyone wouldn’t agree with (independently of what research might say, but based on own experiences). Can we rephrase it like this:

```suggestion
-   We have minimal to no
    slides or lectures, as they are forms of information sharing with
    the lowest rates of learning. Instead we include reading tasks, so 
    you can re-read things you may not
    understand, go at your pace, and take notes, all of which support
    learning. To
```

    help reinforce what you just read, the teacher will briefly
    summarise and emphasise key points after you've read it, which is
    another form of repetition and targets another part of the brain
    (the auditory pathway).

-   We include several breaks throughout the workshop to help you
    recharge and refocus. The brain is only able to effectively focus on
    one thing at a time in 30-50 minute blocks of time. That's why we
    mix together code-alongs, exercises, discussions, and reading to
    give your brain something different to focus on. But having an
    explicit break, even 5 minutes, is enough for the brain to better
    process and store what was covered.

-   We put all the material online so that it is easy for you to go back
    to after the workshop, which builds the repetition and retrieval
    practice part of learning.

-   Our brains can get easily distracted, which is completely normal! So
    if you get distracted during the workshop, you can go to the
    material online to quickly get caught up with where we are. We also

>     make sure to follow the material exactly, so that it is much easier
>     for you to follow along.

Reviewer: ```suggestion
    make sure to follow the material on the website closely, so that it is easier
    for you to follow along.
```

    more relaxed state that also happens to be optimal for learning.
    Hence why we try to explicitly create that space, for example with
    the use of our Code of Conduct.

-   Asking for help can be a hard thing to do, especially in settings
    where you are learning and might be confused or struggling with
    something being taught. You might feel vulnerable asking for help
    and showing that you don't know something. To help reduce these
    feelings and to create a safer space, we regularly get you to put up
    little origami hats (or stickies) to assess how everyone is doing.
    If you need help or are struggling with something, you put up the
    "help" hat (usually orange) and a helper will come to you as soon as
    they can. We often have several helpers, so that you can get help
    quickly. Putting up a little hat is a low effort way of asking for
    help. You don't need to raise your hand that everyone can see nor
    interrupt the teacher, and you can keep trying to follow along (or
    fix the issue) as you wait for a helper to come over. This means it
    will be easier for you to ask for help, which is what we want in a
    learning setting! :grin:

> Now you're aware enough on how we've designed this workshop to really
> support and enable your learning! :tada: :grin:

Reviewer: ```suggestion
Now you know a bit more about the thoughts behind how we've designed this workshop to really try to
support and enable your learning! :tada: :grin:
```
Maybe?

### Other details {.unnumbered}

Aside from the learning design of the workshop, these are some
additional details for the workshop:

-   The schedule listed in @sec-schedule is a *guide* only, some
    sessions are longer, others shorter. It is not intended to be firm
    or strict.

-   We seek out and use feedback to improve the workshops. At the end of
    the workshop, you will be asked to fill out an online survey and
    give your feedback on it. We really want and need honest,
    constructive feedback to improve things! While you are a participant
    of the workshop, you are also a collaborator and "tester" of the
    material. Every time we run a workshop, there are always things to

>     improve on and you help us identify those things.

Reviewer: ```suggestion
    improve on and your feedback help us identify those things.
```
